File: open_quiz_server/quizApp/Models/QuestionAnswerResponseModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace quizApp.Models
{
    public class QuestionAnswerResponseModel
    {
        public bool IsCorrectAnswer { get;  set; }
        public string YourAnswer { get;  set; }
        public int AcomulatedPoints { get;  set; }
        public bool IsGameFinished { get;  set; }
        public string CorrectAnswer { get;  set; }
        public Guid QuestionID { get;  set; }
    }
}

File: open_quiz_client/quizApp/src/app/Models/QuestionAnswerResponseModel.ts
// https://www.npmjs.com/package/guid-typescript
import { Guid } from "guid-typescript";

export interface QuestionAnswerResponseModel {
    isCorrectAnswer: boolean;
    yourAnswer: string;
    acomulatedPoints: number;
    isGameFinished: boolean;
    correctAnswer: string;
    questionID: Guid;
}